Makefile

```diff
@@ -65,12 +89,16 @@ TABLES := reftables
 
 TRM := trm
 
-TXN_DATA := txndata
+TXN_DATA_LON := $(wildcard txndata/txndata-lon/*.lisp) \
+			    $(wildcard txndata/txndata-lon/**/*.lisp)
+
+TXN_DATA_SHAN := $(wildcard txndata/txndata-shan/*.lisp) \
+			     $(wildcard txndata/txndata-shan/**/*.lisp)
 
 WCP := wcp
 
 # Corset is order sensitive - to compile, we load the constants first
-ZKEVM_MODULES := ${CONSTANTS} \
+ZKEVM_MODULES_LONDON := ${CONSTANTS} \
```

Comment:
It would probably be good to have the common modules as their own variable, something `ZKEVM_MODULES_COMMON` rather than repeat them all in both `ZKEVM_MODULES_LONDON` and `ZKEVM_MODULES_SHANGHAI`.

Reply:
FYI: I actioned this already since I need to add the `all` target to the `Makefile`.